docs/contributors/getting-started.md

```diff
@@ -96,3 +96,26 @@ You can launch Storybook by running `npm run storybook:dev` locally. It will ope
 You can also test Storybook for the current `master` branch on GitHub Pages: [https://wordpress.github.io/gutenberg/](https://wordpress.github.io/gutenberg/)
 
 [Storybook]: https://storybook.js.org/
+
+## Developer Tools
+
+It is recommended to configure your editor to automatically check for syntax, and lint errors. This will help you save time as you develop to fix minor formating styles. Here are some directions for setting up Visual Studio Code, a popular editor used by many of the core developers.
```

Comment:
A few minor suggestions on this paragraph:
> We recommend configuring your editor to automatically check for syntax and lint errors. This will help you save time as you develop by automatically fixing minor formatting issues. Here are some directions for setting up Visual Studio Code, a popular editor used by many of the core developers.
1. Eliminating use of passive voice in first sentence.
2. Dropping unnecessary comma near end of first sentence.
3. Fix typo in "formating".
4. Minor sentence structure tweak in second sentence.

Reply:
Thanks :+1: